### DISCUSSION TITLE
Question: updating some models

### DISCUSSION THREAD

**Discussion by ghost** [vladmandic/face-api#150] [General]
**Issue Description**

I am trying to identify and cover / blur faces in IDs, using Node. I have tried the 3 models and the most refined one does well. But I wonder whether we could include some even better trained models for face detection ? 

I have checked for updates for the SsdMobile (the one that worked best) but there aren't afaik. I wouldn't mind to help doing this but am not sure where to start.

It works quite well for clean IDs, but fails for some and particularly IDs that have 2 images one of them being faded or small (for some I got it to also remove both faces.)

Thank you for keeping this project alive, is rather amazing and simple to use to some extent.

**Comment by vladmandic** [comment]
try using my newer project <https://github.com/vladmandic/human> - its far more advanced than `face-api`

btw, i'm moving this issue to discussions as its not a product issue as such, but feel free to continue on the thread.
